api/buildout.cfg

```diff
@@ -50,3 +50,6 @@ site-replace = True
 [versions]
 plone.tiles = 2.0.0b3
 plone.app.mosaic = 2.0.0rc8
+# Those pins are necessary, otherwise buildout will fail with "UnknownExtra: plone.namedfile 4.2.4 has no such extra feature 'blobs'"
+plone.namedfile = 4.2.3
```

Comment:
@thet I have to do those pins in every single 5.1 buildout (on Ubuntu). Any idea what causes this? Brown bag release?